frontend/src/scenes/data-management/event-properties/EventPropertyDefinitionsTable.scss

```diff
@@ -0,0 +1,64 @@
+.event-properties-definition-table {
+    margin-top: 1rem;
```

Comment:
This is a bit off from the rest of our pages, we don't have this extra margin generally (I think Experiments are the closest comparison)